cmd/junitreport/pkg/api/test_case.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"strings"
+	"time"
+)
+
```

Comment:
weird file divisions... why isn't the TestCase type and String impl in this file?